R/endpoints.R

```diff
@@ -49,7 +59,7 @@ endpoint_validate_input <- function(req, res, type, path) {
 #'
 #' @return Formatted hintr response.
 #' @keywords internal
-hintr_response <- function(value, schema) {
+hintr_response <- function(value, schema, auto_unbox = FALSE) {
```

Comment:
I wonder if you have any good ideas for a better way to manage this? So I can't rely on wrapping every single value in `scalar` call in the json. 

Part of the problem is we get a file of json. Read it in which converts it to a list which we then validate. It then gets written back into json when `hintr_response` is called. Perhaps it is better to use the raw json after it has been validated and add an alternative `hintr_response` function for when the data is already in json format.

Reply:
I don't remember if there's an easy way of interpolating json, but I don't think we should be roundtripping it if we don't need to